src/cclib/parser/daltonparser.py

```diff
@@ -166,7 +174,8 @@ def extract(self, inputfile, line):
             symmetry_atoms = []
             atommasses = []
             for cols in lines:
-                atomnos.append(self.table.number[cols[0]])
+                cols0 = ''.join([i for i in cols[0] if not i.isdigit()]) #remove numbers
```

Comment:
OK. Maybe they were added to cclib/master in the meantime... could you rebase your pull request?